src/components/rating/RatingStar.tsx

```diff
@@ -0,0 +1,31 @@
+import { ComponentProps, FC } from 'react';
+import classNames from 'classnames';
+import { useRatingContext } from './RatingContext';
+
+export type RatingStarProps = ComponentProps<'svg'> & {
+  filled?: boolean;
+};
+
+const sizeClasses = {
+  sm: 'w-5 h-5',
+  md: 'w-7 h-7',
+  lg: 'w-10 h-10',
+};
+
+export const RatingStar: FC<RatingStarProps> = ({ filled = true }) => {
+  const { size } = useRatingContext();
+
+  return (
+    <svg
```

Comment:
You can use `HiIcon` from the `react-icons` library it is the same one as this, I checked